app/src/main/java/com/khinthirisoe/bakingapp/ui/baking/view/BakingViewHolder.kt

```diff
@@ -0,0 +1,35 @@
+package com.khinthirisoe.bakingapp.ui.baking.view
+
+import android.view.View
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Recipe
+import com.khinthirisoe.bakingapp.ui.base.BaseViewHolder
+import com.khinthirisoe.bakingapp.ui.ingredients.IngredientsActivity
+import kotlinx.android.synthetic.main.list_baking.view.*
+
+class BakingViewHolder(itemView: View) : BaseViewHolder(itemView) {
+
+    fun onBind(recipe: Recipe, position: Int) {
+
+        itemView.txt_no_of_serving.text = recipe.servings.toString()
+        itemView.txt_baking_name.text = recipe.name
+
+        itemView.img_baking.setImageDrawable(itemView.context.resources.getDrawable(bakingImages[position]))
```

Comment:
@KhinThiriSoe leave comment about the relation between the position and the images.
For example why position 0 is nutella_pie and 1 is brownies

Reply:
just make it depends on json response :D

Reply:
@KhinThiriSoe what if the position in the json response changed?

Reply:
hee. no idea yet.

Reply:
@KhinThiriSoe recipe class has `image` field. Is it the same as the image you're using here?

Reply:
No, json has just only image field and no data.

Reply:
@KhinThiriSoe when there's a new recipe, you'll have arrayIndexBoundException. Fix that

Reply:
fixed it :)